djangoTrading/settings.py

```diff
@@ -11,22 +11,26 @@
 """
 
 from pathlib import Path
+from dotenv import load_dotenv
+import os
+
+from celery.schedules import crontab
+
+load_dotenv()
 
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
 
-
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/3.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'django-insecure-0z=7ybt!dh85py0=id=r=78r#*6*c7y9cd2*gr)7q$*j^d2^bt'
+SECRET_KEY = str(os.getenv('SECRET_KEY'))
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
-
-ALLOWED_HOSTS = []
+DEBUG = str(os.getenv('DEBUGMODE', True))
```

Comment:
it looks strange. Maybe i've made a mistake was describes it to you. Even if it will be False, debug will True, because string is not empty.
The better way is:
.env file:
```ini
DEBUG_MODE=true
```
settings.py:
```python
DEBUG = os.environ.get('DEBUG_MODE', 'true').lower() == 'true'
```